drafts/Blockchain Based Lease Agreement Management Tool on Fuel Network/README.md

```diff
@@ -0,0 +1,88 @@
+Also the fraud proof indirectly causes the lower costs. It assumes transactions are valid by default but allows anyone to challenge them later if they're fraudulent. This eliminates the need for everyone to verify every transaction, reducing the load on the network.
+
+5.	Problem of double spending 
+Fuel relies on a system called the Unspent Transaction Output (UTXO) model. Imagine each coin on the network has a unique fingerprint. To spend a coin, you need to prove you own it by showing where it came. This makes it very hard to fake ownership or spend a coin you don't have.
+
+6.	Development Level Features
+Fuel network comes with sway language which targets FuelVM which also a powerful tool for development. Developers can apply the advantages of parallel transaction execution and fraud-proofing capabilities to Lease Agreement Management System.
+
+## Conclusion
+In conclusion, the adoption of blockchain technology in Lease Agreement Management offers numerous advantages, including increased transparency, reduced fraud risks, improved transaction efficiency, and enhanced security. However, several challenges such as scalability, security concerns, and regulatory frameworks need to be addressed for widespread implementation. The Fuel Network presents a promising solution to many of these challenges with its powerful architectural features. By leveraging public ledger technology and unique mechanisms like UTXO fraud proofs, modular execution and predicates, Fuel enhances transparency, security, and scalability in Lease Agreement Management Applications. Furthermore, its modular design and development tools provide developers with the flexibility and efficiency needed to build robust and secure Lease Agreement Management Systems. Overall, the Fuel Network demonstrates great potential in revolutionizing Lease Agreement Management practices and addressing the key concerns faced by traditional systems and tools based on Ethereum network. 
+
+## Sample Lease Agreement Management Application Based on Fuel.
+Following tutorial demonstrates how to apply capabilities and tools on fuel network to address these challenges.
+
+With this tutorial, we will develop a full-stack DApp that has the following features which can demonstrate some fundamental functions of lease business.
+
+1. Land Lords can initiate a lease
+2. Tenants can view the status of the lease
+3. Tenants can pay lease
+4. Tenants can request for termination or renewal
+
+### Technology Used
+
+**Sway**: Rust based programming language and also the domain specific language for FuelVMs 
+**Fuels**: A fully purpose-built, custom virtual machine for executing smart contracts   
+**Fuel Typescript SDK** : SDK to interact with fuel contracts 
+**Angular**: A popular typescript based frontend framework  
+                                               
+-----
+#### Let's Begin!
```

Comment:
Hey @RandulaKoralage thank you for this article. Before commenting on other aspects of it, I think it's missing the content for this section, which is the tutorial walking the reader through creating the contract and frontend.